operator/cmd/flags.go

```diff
@@ -326,6 +326,9 @@ func init() {
 	flags.StringSlice(operatorOption.IngressLBAnnotationPrefixes, operatorOption.IngressLBAnnotationsDefault, "Annotation prefixes for propagating from Ingress to the Load Balancer service")
 	option.BindEnv(Vp, operatorOption.IngressLBAnnotationPrefixes)
 
+	flags.Bool(operatorOption.EnableK8s, true, `Enable operation of kubernetes-related services/controllers when using cilium w/ kubernetes`)
```

Comment:
```suggestion
	flags.Bool(operatorOption.EnableK8s, true, `Enable operation of Kubernetes-related services/controllers when using Cilium with Kubernetes`)
```

Reply:
Hi, I've pushed an updated branch with this fixes applied.